README.md

```diff
@@ -31,11 +31,26 @@ support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testi
 * [2024/07] Performance optimizations and usability improvements on [client-side](https://github.com/intel/neural-compressor/blob/master/docs/source/3x/client_quant.md).
 
 ## Installation
+### Install Framework
+#### Install torch for CPU
+```Shell
+pip install torch --index-url https://download.pytorch.org/whl/cpu
+```
+#### Install torch/intel_extension_for_pytorch for XPU
+https://intel.github.io/intel-extension-for-pytorch/index.html#installation 
+
```

Comment:
how about Gaudi?,  Should we also add one section for Gaudi to let user know torch has already in its official docker image and no need to install separately?

Reply:
#### Use Docker Image with torch installed for HPU
```Shell
docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice --net=host --ipc=host vault.habana.ai/gaudi-docker/1.16.2/{$OS}/habanalabs/pytorch-installer-2.2.2:latest   # this cmd should be updated after 1.17 release. 
```
https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#bare-metal-fresh-os-single-click